docs/machine-learning/resources/transforms.md

```diff
@@ -0,0 +1,131 @@
+| FeatureSelectorByMutualInformation | Selects the top k slots across all specified columns ordered by their mutual information with the label column. |
+
+## Featurizers
+
+| Transform | Definition |
+| --- | --- |
+| HashConverter	| Converts column values into hashes. This transform accepts both numeric and text inputs, both single and vector-valued columns. This is a part of the Dracula transform. |
+| TreeLeafFeaturizer | Trains a tree ensemble, or loads it from a file, then maps a numeric feature vector to three outputs: 1. A vector containing the individual tree outputs of the tree ensemble. 2. A vector indicating the leaves that the feature vector falls on in the tree ensemble. 3. A vector indicating the paths that the feature vector falls on in the tree ensemble. If a both a model file and a trainer are specified - will use the model file. If neither are specified, will train a default FastTree model. This can handle key labels by training a regression model towards their optionally permuted indices. |
+
+## Label parsing
+
+| Transform | Definition |
+| --- | --- |
+| Dictionarizer	| Converts input values (words, numbers, etc.) to index in a dictionary. |
+| LabelColumnKeyBooleanConverter | Transforms the label to either key or bool (if needed) to make it suitable for classification. |
+| LabelIndicator | Label remapper used by OVA. |
+| LabelToFloatConverter	| Transforms the label to float to make it suitable for regression. |
+| PredictedLabelColumnOriginalValueConverter | Transforms a predicted label column to its original values, unless it is of type bool. |
+ 
+## Missing Values 
+           
+| Transform | Definition |
+| --- | --- |
+| MissingValueHandler | Handle missing values by replacing them with either the default value or the mean/min/max value (for non-text columns only). An indicator column can optionally be concatenated, if theinput column type is numeric. |
+| MissingValueIndicator	| Create a boolean output column with the same number of slots as the input column, where the output value is true if the value in the input column is missing. |
+| MissingValuesDropper | Removes NAs from vector columns. |
+| MissingValuesRowDropper | Filters out rows that contain missing values. |
+| MissingValueSubstitutor | Create an output column of the same type and size of the input column, where missing values are replaced with either the default value or the mean/min/max value (for non-text columns only). |
+
+## Normalization
+
+| Transform | Definition |
+| --- | --- |
+| BinNormalizer	| The values are assigned into equidensity bins and a value is mapped to its bin_number/number_of_bins. |
+| ConditionalNormalizer	| Normalize the columns only if needed. |
+| GlobalContrastNormalizer | Performs a global contrast normalization on input values: Y = (s * X - M) / D, where s is a scale, M is mean and D is either L2 norm or standard deviation. | 
```

Comment:
comma between "mean" and "and"
L2 norm or standard deviation --> the L2 norm or the standard deviation